src/Serilog.Extensions.Hosting/Extensions/Hosting/DiagnosticContextCollector.cs

```diff
@@ -38,19 +39,59 @@ public void AddOrUpdate(LogEventProperty property)
             }
         }
 
+        /// <summary>
+        /// Set an exception to include in the Serilog request log event.
```

Comment:
Nit: _Serilog.Extensions.Hosting_ is independent of the HTTP functionality in _Serilog.AspNetCore_; may be better to keep this simple - perhaps just "Set the exception associated with the current diagnostic context.` or similar, and keep the other information in example/remarks?